happi/client.py

```diff
@@ -10,7 +10,6 @@
 ###############
 # Third Party #
 ###############
```

Comment:
This is why I don't like the import comment headers. Look at the big "we don't use 3rd party modules" banner it leaves. https://www.python.org/dev/peps/pep-0008/#imports specifies only a blank line between each import group.

Reply:
I'll remove these in the next PR